app/code/local/Excellence/Test/controllers/IndexController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+class Excellence_User_IndexController extends Mage_Core_Controller_Front_Action
+{
+    public function deleteAction()
+    {   
+        if( $this->getRequest()->getParam('user_id') > 0 ) {
+         try {
+              $user_model = Mage::getModel('user/user');
+              $address_model = Mage::getModel('user/address');
+              $user_model->setId($this->getRequest()->getParam('user_id'))->delete();
+              $address_model->setId($this->getRequest()->getParam('user_id'))->delete();
```

Comment:
does delete work? because user_id cannot be same for both user_model and address_model